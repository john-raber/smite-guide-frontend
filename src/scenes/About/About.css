/* Setting the background color to match the App's background */
.About {
  background-color: rgb(72, 76, 84);
  text-align: left;
}

/* Setting the font color for the title and paragraphs on the about page */
.About p,
h3 {
  color: rgb(255, 255, 255);
}

/* Setting the color of the horizontal line on the about page */
.About hr {
  border-color: rgb(255, 255, 255);
}
