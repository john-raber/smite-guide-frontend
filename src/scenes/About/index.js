import React from "react";
import { withRouter } from "react-router-dom";
import { Container, Jumbotron } from "reactstrap";

import "./About.css";

const About = () => {
  return (
    <Jumbotron className="About">
      <h3 className="display-3">About</h3>
      <hr className="my-2" />
      <p>
        After playing Smite for years, experienced players have a level of
        understanding of what each god in the game can do and what gods can
        counter a specific god's abilities. I have tried to get some of my
        friends to get into the game and the biggest challenge in getting them
        to play is the knowledge barrier.
      </p>
      <p>
        There are currently 98 gods in Smite and that number is constantly
        growing, so the knowledge gap between experienced players and someone
        who is just picking up the game is constantly growing. SmiteGuide is a
        site where experienced Smite players can share their knowledge of the
        game with those who are just getting started and learning the game.
      </p>
    </Jumbotron>
  );
};

export default withRouter(About);
