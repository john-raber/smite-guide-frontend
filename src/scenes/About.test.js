import React from "react";
import { shallow } from "enzyme";

import { About } from "./About/index";

describe("About", () => {
  it("renders correctly", () => {
    let wrapper = shallow(<About />);

    expect(wrapper.find(".About").length).toEqual(1);
  });

  it("matches its snapshot", () => {
    let wrapper = shallow(<About />);

    expect(wrapper).toMatchSnapshot();
  });
});
