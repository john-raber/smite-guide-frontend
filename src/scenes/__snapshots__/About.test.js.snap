// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`About matches its snapshot 1`] = `
<Jumbotron
  className="About"
  tag="div"
>
  <h3
    className="display-3"
  >
    About
  </h3>
  <hr
    className="my-2"
  />
  <p>
    After playing Smite for years, experienced players have a level of understanding of what each god in the game can do and what gods can counter a specific god's abilities. I have tried to get some of my friends to get into the game and the biggest challenge in getting them to play is the knowledge barrier.
  </p>
  <p>
    There are currently 98 gods in Smite and that number is constantly growing, so the knowledge gap between experienced players and someone who is just picking up the game is constantly growing. SmiteGuide is a site where experienced Smite players can share their knowledge of the game with those who are just getting started and learning the game.
  </p>
</Jumbotron>
`;
